ui-v2/app/components/copy-button.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@ember/component';
+import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+import config from 'consul-ui/config/environment';
+
+import WithListeners from 'consul-ui/mixins/with-listeners';
+
+export default Component.extend(WithListeners, {
+  clipboard: service(`clipboard/${config.environment === 'test' ? `local-storage` : `os`}`),
```

Comment:
What do you think about overriding the `clipboard/os` service in tests rather than having the ternary here?

Reply:
Definitely would prefer that myself also. I wasn't immediately aware how to inject services just for acceptance tests though, and I didn't want to get into potential 🐰 🕳 's looking into that. To be honest though I didn't even start looking into it and just went with this as a 'good enough' thing. I'll investigate a little more today though, and come back here. If I don't get very far though maybe you can help me with it? Will shout you later.